src/main/resources/com/flowdock/jenkins/FlowdockNotifier/config.jelly

```diff
@@ -16,10 +21,18 @@
          title="Test connection with Chat notification" progress="Sending..."
          method="testConnection" with="flowToken,notificationTags" />
 
+      <f:entry title="Chat notification when build starts" field="chatStart">
```

Comment:
😍

Reply:
I mean, nice to maker a field on the config form, but it didn't actually do anything.